packages/components/Button/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/button",
-  "version": "0.32.3",
+  "version": "0.32.4",
```

Comment:
NPM says the last published is 0.32.3
https://www.npmjs.com/package/@fluentui-react-native/button